libs/sdk-flutter/lib/breez_sdk.dart

```diff
@@ -113,9 +87,8 @@ class BreezSDK {
   Future connect({
     required ConnectRequest req,
   }) async {
-    await _lnToolkit.connect(
-      req: req,
-    );
+    await _lnToolkit.connect(req: req);
+    _subscribeToSdkStreams();
```

Comment:
I don't think this will work atm. The rust side only accept one registration in a lifetime. The second time connect will be called it will fail.

Reply:
We have a single registration on initialize, which passes a copy of the value that references the SDK's stream to instance's Stream:
```
 // Flutter Instance Stream
Stream<BreezEvent>? _breezEventsStream;

void _initializeEventsStream() {
    // Stream from Rust
    _breezEventsStream = _lnToolkit.breezEventsStream(); 
}
```

which we can subscribe & unsubscribe from via `StreamSubscription<BreezEvent>? _breezEventsSubscription;`

I will test it today.

Reply:
@roeierez Works as expected.

Made a required change to be able to listen the streams multiple times: https://github.com/breez/breez-sdk/pull/980/commits/6f52b18e3474e5cd5787feaf5b03b3ac9b5a2af1